ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -332,24 +333,30 @@ export const WorkspaceShare = withCurrentWorkspace()(class extends React.Compone
       <Modal width={450}>
         {this.state.saving && <SpinnerOverlay />}
         <ModalTitle style={styles.modalTitle}>
-          <div>
-          <label>Share {this.props.workspace.name}</label>
-          <TooltipTrigger content={<div style={styles.tooltipLabel}>
-            Search for a collaborator that has an <i>All of Us</i> research account. Collaborators can
-            be assigned different permissions within your Workspace.
-            <ul>
-              <li style={styles.tooltipPoint}>A <span style={{'textDecoration': 'underline'}}>
-                Reader</span> can view your notebooks, but not make edits, deletes or share
-                contents of the Workspace.</li>
-              <li style={styles.tooltipPoint}>A <span style={{'textDecoration': 'underline'}}>
-                Writer</span> can view, edit and delete content in the Workspace but not share
-                the Workspace with others.</li>
-              <li style={styles.tooltipPoint}>An <span style={{'textDecoration': 'underline'}}>
-                Owner</span> can view, edit, delete and share contents in the Workspace.</li>
-            </ul>
-          </div>}>
-            <InfoIcon style={{width: '13px', height: '14px', marginLeft: '.4rem'}}/>
-          </TooltipTrigger>
+          <FlexRow style={{alignItems: 'center', justifyContent: 'space-between'}}>
+            <div>
+              <label>Share {this.props.workspace.name}</label>
+            </div>
+            <TooltipTrigger content={<div style={styles.tooltipLabel}>
+              Search for a collaborator that has an <i>All of Us</i> research account. Collaborators can
+              be assigned different permissions within your Workspace.
+              <ul>
+                <li style={styles.tooltipPoint}>A <span style={{'textDecoration': 'underline'}}>
+                  Reader</span> can view your notebooks, but not make edits, deletes or share
+                  contents of the Workspace.</li>
+                <li style={styles.tooltipPoint}>A <span style={{'textDecoration': 'underline'}}>
+                  Writer</span> can view, edit and delete content in the Workspace but not share
+                  the Workspace with others.</li>
+                <li style={styles.tooltipPoint}>An <span style={{'textDecoration': 'underline'}}>
+                  Owner</span> can view, edit, delete and share contents in the Workspace.</li>
+              </ul>
+            </div>}>
+              <InfoIcon style={{width: '14px', height: '14px', marginLeft: '.4rem'}}/>
+            </TooltipTrigger>
+          </FlexRow>
+          <div style={{color: colors.primary, fontSize: 14, fontWeight: 400}}>
```

Comment:
[non-opt] The ticket description requires that the share modal content changes depending on the free-tier status of the workspace. This requires some add'l logic based on workspace state, no?

Reply:
Aha! I missed this, good catch. Will change.